### PR TITLE
feat(Split button): Allow texts in split button

### PR DESCRIPTION
**Pull request by karelhala**
Fixes: #2758

**What**:
Allows passing children to split button in order to allow passing number of selected to this component.

**Additional issues**:

### CODE REVIEW

**Comment by karelhala** [comment]
I see that there is also PR https://github.com/patternfly/patternfly-react/pull/2838 for version bump. I'll wait for it to merge and rebase.

**Comment by mcoker** [comment]
@mcarrano good catch, that's because there is a space character in the HTML example and no space in the react example. I don't think we want to rely on a space being there. I can submit a PR to address this in core and we can pull that in for this PR.

**Comment by mcoker** [comment]
@karelhala if you can update to >= 2.31.3, the spacing should be fixed via https://github.com/patternfly/patternfly-next/issues/2223

Also looks like there are conflicts now.

**Comment by karelhala** [comment]
@mcarrano @mcoker good catch! And thank you for the new styles update. I updated PF core to newest version, can you check if it looks good now?

**Comment by mcarrano** [comment]
@karelhala @mcoker When I look at this in Firefox, looks like the alignment of the checkbox and text are a bit off.

Chrome:

Screen Shot 2019-09-06 at 9 13 33 AM

Firefox:

Screen Shot 2019-09-06 at 9 13 24 AM

**Comment by mcoker** [comment]
@mcarrano checkboxes can be hard to style consistently cross-browser. We ran into this with the check component before, and I see the same thing now on that component

Screen Shot 2019-09-06 at 8 42 09 AM

Feel free to open an issue if you'd like us to look at it again. Using custom checkboxes should solve this.

File: packages/patternfly-4/react-inline-edit-extension/package.json [hunk 1]
```diff
@@ -27,9 +27,15 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/",
   "dependencies": {
+<<<<<<< HEAD
```
Comment (redallen) [inline packages/patternfly-4/react-inline-edit-extension/package.json]: You committed merge conflicts.

Comment (karelhala) [reply packages/patternfly-4/react-inline-edit-extension/package.json]: Ah, thanks for noticing!

File: packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx [hunk 2]
```diff
@@ -16,6 +16,8 @@ export interface DropdownToggleCheckboxProps extends Omit<React.HTMLProps<HTMLIn
   checked?: boolean | null;
   /** A callback for when the checkbox selection changes */
   onChange?(checked: boolean, event: React.FormEvent<HTMLInputElement>): void;
+  /** */
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx]: Add comment: `Element to be rendered inside the <span>`

File: packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx [hunk 3]
```diff
@@ -49,6 +58,7 @@ export class DropdownToggleCheckbox extends React.Component<DropdownToggleCheckb
           disabled={isDisabled}
           defaultChecked={isChecked || checked}
         />
+        {text}
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx]: You can just put the whole statement here:

```
children && (
<span
  className={css(styles.dropdownToggleText, className)}
  aria-hidden="true"
  id={`${props.id}-text`}
>
  {children}
</span>
)
```

Comment (karelhala) [reply packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx]: Yeah, I did that and the TS lint complained about `jsx-no-multiline-js` https://github.com/palantir/tslint-react

File: packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx (hunk 2 above)
Comment (karelhala) [reply packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx]: Yeah, good point. Missed that.

File: packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx (hunk 3 above)
Comment (redallen) [reply packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx]: I think you need the `(` and `)` to comply with that linting rule in the future. I'm fine with it either way.

**Pull request merged**